packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -35,6 +35,16 @@ type SelectInputProps = Pick<
   | 'testID'
 > & {
   icon?: IconComponent;
+  /**
+   * Controlled value of the Select. Use it in combination of `onChange`.
+   *
+   * Check out [Controlled Dropdown Documentation](https://blade.razorpay.com/?path=/story/components-dropdown-with-select--controlled-dropdown&globals=measureEnabled:false) for example.
+   */
+  value?: string | string[];
+  /**
+   * Set initial value of Select. Use `value` instead if you want to make Dropdown controlled.
```

Comment:
Probably below description? this is what we use in text input.
Used to set the default value of Select Input field when it's uncontrolled